api/hasura/event_triggers/eventManager.ts

```diff
@@ -25,6 +27,8 @@ const HANDLERS: HandlerDict = {
   createNomineeTelegram,
   createCircleCRM,
   createVouchedUser,
+  interactionEventToAmplitude,
+  interactionEventToMixpanel,
```

Comment:
```suggestion
  sendEventToAmplitude,
  sendEventToMixpanel,
```

just a suggestion, since the convention here is to start with a verb

Reply:
renamed